api-schema/src/main/java/org/stellar/anchor/api/sep/sep6/InfoResponse.java

```diff
@@ -77,13 +78,16 @@ public static class DepositAssetResponse {
     @SerializedName("max_amount")
     Long maxAmount;
 
+    @SerializedName("funding_methods")
+    List<String> fundingMethods;
+
     /**
      * The fields required to initiate a deposit.
      *
      * <p>The only field supported by the platform is <code>type</code>. Additional fields required
      * for KYC are supplied asynchronously through SEP-12 requests.
      */
-    Map<String, AssetInfo.Field> fields;
+    @Deprecated Map<String, AssetInfo.Field> fields;
```

Comment:
Can you point me to where it's missing ? I believe I have marked it as deprecated

Reply:
```suggestion
    /**
     * Deprecated since SEP-6 version 4.1.0.
     * https://github.com/stellar/stellar-protocol/blob/21d30cd22caa6f25369828289d845c4366eaa641/ecosystem/sep-0006.md
     */
    @Deprecated Map<String, AssetInfo.Field> fields;
```